google/cloud/forseti/services/cli.py

```diff
@@ -25,10 +25,26 @@
 from google.protobuf.json_format import MessageToJson
 
 from google.cloud.forseti.services import client as iam_client
+from google.cloud.forseti.common.util import file_loader
 from google.cloud.forseti.common.util import logger
 
 LOGGER = logger.get_logger(__name__)
 
+
+class DefaultParser(ArgumentParser):
+    """Default parser, when error is triggered, instead of printing
+    error message, it will print the help message (-h).
+    """
+    def error(self, message):
```

Comment:
We are creating a wrapper class on top of the ArgumentParser class so that when error occurs, we catch them and print the usage instead of the error message.